lib/Logger.ts

```diff
@@ -127,6 +128,11 @@ class Logger {
   public debug = (msg: string) => {
     this.log(Level.DEBUG, msg);
   }
+
+  public silly = (msg: string) => {
```

Comment:
So I know the underlying winston level is `SILLY`, but I might prefer calling this method `trace` for our own purposes. `logger.trace()` sounds more sensible to me than `logger.silly()`. Thoughts?